### PR TITLE
Add update asset tutorial

### PR DESCRIPTION
**Pull request by edd**
Adds a new page for the updateAsset command

- Add updateAsset examples (untested)
- Fix network paramter display
- Fix more network param formatting in other proposals
- Fix update asset to include quantum

### CODE REVIEW

**Comment by edd** [comment]
@candida-d The documentation is possibly a bit basic - but there’s also not a lot to say. You can update very few fields. But if you want more in here, let me know,

File: docs/tutorials/proposals/update-asset-proposal.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 4
+title: Update an asset
```
Comment (candida-d) [inline docs/tutorials/proposals/update-asset-proposal.md:3]:
```suggestion
title: Propose asset update
```

File: docs/tutorials/proposals/update-asset-proposal.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import UpdateAssetAnnotated from './_generated-proposals/_updateAsset_annotated.md';
+import UpdateAssetJSON from './_generated-proposals/_updateAsset_json.md';
+import UpdateAssetCMD from './_generated-proposals/_updateAsset_cmd.md';
+import UpdateAssetWin from './_generated-proposals/_updateAsset_win.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Propose an update to an asset
+
+Some of the properties on an asset can be changed through governance. Important fields such as the underlying contract, name and symbol cannot be changed.
```
Comment (candida-d) [inline docs/tutorials/proposals/update-asset-proposal.md:27]:
```suggestion
Some of the properties of an asset can be changed through governance. Those fields are: withdrawal and deposit limits, and the asset's quantum.

The underlying contract, asset name and symbol cannot be changed.
```

**Review by candida-d — COMMENTED** [review]
I made some copy suggestions - but I believe updating assets requires getting changes into the contract itself, and if so that needs to be mentioned.

**Comment by edd** [comment]
@candida-d I’ll add a mention of the bridge update to this and create (if it’s not already there), and then move on to #234 to actually get that part documented

**Review by candida-d — APPROVED** [review]

**Pull request merged**